lang/es.json

```diff
@@ -30,7 +30,7 @@
   "Seek to live, currently behind live": "Buscar en vivo, actualmente demorado con respecto a la transmisión en vivo",
   "Seek to live, currently playing live": "Buscar en vivo, actualmente reproduciendo en vivo",
   "Progress Bar": "Barra de progreso",
-  "progress bar timing: currentTime={1} duration={2}": "{1} de {2}",
+  "progress bar timing: currentTime={1} duration={2}": "Barra de progeso: Transcurrido{1} de {2}",
```

Comment:
This was correct as "{1} de {2}". This one is a bit of a special case, as it's a translation with variable substitution. In English, it becomes "{1} of {2}".

Reply:
It is to make the translation look better

Reply:
In addition, a screen reader trying to try to read out that entire phrase will fall behind by quite a lot compared to it already likely falling being just reading `{1} of {2}`.
The translations should match what's here https://github.com/videojs/video.js/blob/main/lang/en.json#L17

Reply:
@segus3088 if you can revert this line change we'd be happy to merge it in. Thanks!

Reply:
@gkatsev look better "Barra de progreso: {1} de {2}"